stdlib/cmake/modules/SwiftSource.cmake

```diff
@@ -656,6 +656,12 @@ function(_compile_swift_files
            "-emit-private-module-interface-path" "${private_interface_file}")
     endif()
 
+    if(SWIFT_STDLIB_EMIT_API_DESCRIPTORS AND NOT SWIFTFILE_IS_FRAGILE)
+      set(api_descriptor_file "${module_base}.partial.sdkdb")
```

Comment:
BTW, it would be nice to rename this to `${module_base}.partial.json`.  I don't think that sdkdb is a great extension to use here for the JSON blob.

Reply:
This is addressed in 10dbd8bb80de167027d673edfaf36b75ad62640f